src/services/api/userApi.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { ax } from '../axios';
+import { UserDataResponse } from '@/models/user/userApiModels';
+
+export const userData = async () => {
```

Comment:
반환 타입을 명시적으로 지정하는 것이 좋을 것 같습니다.
```suggestion
export const userData = async (): Promise<UserDataResponse> => {
```

Reply:
✅ be677dccef09776d5a85f0a6c5075f3856bd58ec